### PR TITLE
Add support for a manifest auth token.

### PR DESCRIPTION
**Pull request by tpendragon**
This will allow us to have the system act as an authorized user when
indexing.

### CODE REVIEW

**Comment by tpendragon** [comment]
Closing without merge, it's not passing it through for the exhibit.

File: app/models/collection_manifest.rb [hunk 1]
```diff
@@ -10,7 +10,9 @@ def self.find_by_slug(slug)
   end
 
   def id
```
Comment (hackmastera) [inline app/models/collection_manifest.rb]: I'm not sure about redefining this method this way. The ID doesn't properly include the auth token, right? It's just that most or all of our uses will require the token. Is it impossible to adjust use without defining it at this level? Are there issues with upstream libraries? I'm concerned that even if there aren't unintended consequences of this now there might be down the road.

This bit of code to add an auth token to a url is repeated several times such that I'm thinking we might want to pull it out into a utility method somewhere. Then we could add a comment there as to its purpose / use.

Comment (tpendragon) [reply app/models/collection_manifest.rb]: I had the same concern and I did a -lot- of testing. This makes the reader output the token, but still persists fine. This ID is used implicitly all over the Spotlight codebase, so this is the smallest overwrite I could find.

Comment (hackmastera) [reply app/models/collection_manifest.rb]: What do you mean it makes the reader output the token?

Comment (tpendragon) [reply app/models/collection_manifest.rb]: `#id` output will include the token, but it won't get saved to the DB. I'll extract a utility method.

**Comment by tpendragon** [comment]
@hackmastera Take another look?

File: app/models/collection_manifest.rb (hunk 1 above)
Comment (hackmastera) [reply app/models/collection_manifest.rb]: Can you add this as a comment and include a canonical link to where it's persisted?

Comment (tpendragon) [reply app/models/collection_manifest.rb]: Added some comments.

**Pull request merged**